### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by susanbuckley**
Took out KB article

### CODE REVIEW

**Comment by nconforti93** [comment]
@susanbuckley  I suggest to create a doc folder and put the style guide in there as well as the templates. I think it's important that the README mentions the overall layout of the repo, how to contribute and work with it, where to find further information, and any restrictions to keep in mind. Adding the very detailed style guide on this level might make it difficult for people to see just how to use it in general

**Review by nconforti93 — CHANGES_REQUESTED** [review]

**Comment by nconforti93** [comment]
Of course, if this isn't necessary you think then we just need to restructure the readme a bit

**Comment by susanbuckley** [comment]
Per one of the Readme or both I thought only you could create

Get Outlook for Android<https://aka.ms/AAb9ysg>

Exasol INC.
Susan Buckley | Technical Writer


***@***.******@***.***>
www.exasol.com<http://www.exasol.com/en/>

Get the latest news:
www.exasol.com/newsroom/<http://www.exasol.com/en/company/newsroom/>

[cid:sign_1-Main_b6957fc3-66b6-468b-b929-70b78e51ebab.png]<https://www.linkedin.com/company/exasol-ag/>[cid:sign_2_twitter_e3a8df5e-6f92-4546-97f2-cfae237d9dae.png]<https://twitter.com/ExasolAG>[cid:sign_3_linkedin_df799929-e125-45ad-afa8-1522692489e1.png]<https://www.linkedin.com/company/exasol-ag/><https://twitter.com/ExasolAG><https://www.xing.com/company/exasol><https://www.linkedin.com/company/exasol-ag/>

This e-mail may contain confidential information. If you are not the intended recipient please notify the sender immediately. Any unauthorised
copying or distribution of this e-mail is forbidden.

________________________________

File: README.md [hunk 1]
```diff
@@ -1,7 +1,122 @@
 # Exasol Knowledgebase
 
-Welcome to the Exasol Knowledgebase! This repository contains source files for Exasol's knowledgebase. The files in this repository are used to create the Knowledgebase, which is accessible at a different link (coming soon!).
+## Welcome
 
+Welcome to the Exasol Knowledgebase! This repository contains source files for Exasol's knowledgebase. The files in this repository are used to create the Knowledgebase, which is accessible at [Exasol.com](https://Exasol.com).
```
[the remark this answers is not in the document]
Comment (susanbuckley) [reply README.md]: Made changes per our meeting this morning.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,122 @@
...
+### upgrade
+
+Do not use with Exasol Database products. Use update or migration.
+
+Use with SaaS when changing subscriptions. For example: "When you upgrade from Standard edition to Enterprise edition…"
+
+### update
+
+The process of installing intermediate releases such as bug fixes. For example, Exasol 7.0 to Exasol 7.1 or Exasol 7.1 to Exasol 7.1.1. products.
 If you wish to contribute to the knowledgebase, feel free to fork the repository and make a pull request with your changes.
```
[the remark this answers is not in the document]
Comment (susanbuckley) [reply README.md]: Made changes per our meeting this morning.

**Comment by susanbuckley** [comment]
@nconforti93 I believe I've made the needed changes and resolved the conflict.

**Review by nconforti93 — APPROVED** [review]
approved

**Pull request merged**